auslib/test/test_db.py

```diff
@@ -2024,7 +2041,7 @@ def testGetRulesMatchingQueryReturnBackgroundThrottled(self):
     def testGetRulesMatchingQueryOsVersionSubstring(self):
         rules = self.paths.getRulesMatchingQuery(
             dict(product="a", version='5.0', channel="a", buildTarget='d',
-                 buildID='', locale='', osVersion='foo 1.2.3', distribution='',
+                 buildID='', locale='', osVersion='foo 1', distribution='',
```

Comment:
I reverted this and two other tests that made use of substring matching, saved me a lot of time in testing whether substring matching works correctly.